Answers.md

```diff
@@ -1,3 +1,13 @@
-### Your responses to the short answer questions should be laid out here using markdown.
+#Answers
 
-For help with markdown syntax, [go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+1) A docker container runs on a bare minimum linux environment which sets on top of a normal OS.  Containers cut out a big chunk of overhead of a typical VM.  Since containers have smaller file size they can be ftp-ed easily over a network, along with being executed quickly in easy commands (e.x. docker run).
+
+2) the -p tells what port to access your app on your container. <host port>:<container port>
+
+3) It helps analyize/monitor/schedule container and instances.  Configure container settings.  Start and stop containers.  Scale them with more ease.
+
+4) Looks like you can specify it in the yaml file?
```

Comment:
To scale a Kubernetes cluster you need to edit the config files and update the `replicas` value. Once you've done that, run the `apply` command from kubernetes cli with the updated file.